frontend/.env

```
@@ -1,2 +1,3 @@
 UMAMI_SITE_CODE="b18c2215-9ec9-4393-a2e5-3eea30aa49a4"
 UMAMI_API_URL="https://cloud.umami.is/script.js"
+PUBLIC_SANITY_PROJECT_ID="sb0cunx0"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)